frontend/src/components/Utils/filter-input.js

```diff
@@ -15,31 +14,32 @@ export function getCleanedTextInput(rawInput, defaultValue) {
 }
 
 /**
- * Return an array of collaborator uids given the emails provided in the
- * add trip form.
+ * Return a promise containing an array of collaborator uids given the emails
+ * provided in the add trip form.
  *
  * TODO(#72 & #67): Remove 'remove empty fields' once there is better way to
  * remove collaborators (#72) and there is email validation (#67).
  *
- * @param {!Array{string}} collaboratorEmailsArr Array of emails corresponding
+ * @param {!Array<string>} collaboratorEmailsArr Array of emails corresponding
  *     to the  collaborators of the trip (not including the trip creator email).
- * @return {!Array{string}} Array of all collaborator uids (including trip
+ * @return {Promise<!Array<string>>} Array of all collaborator uids (including trip
```

Comment:
`string[]` instead of `Array.<string>`?

If not, then I believe the correct jsdoc format is `Array.<string>` (with the dot)

Reply:
This is fixed in #102 with all the changes but for now I think this is technically correct and the current version of that syntax according to[ these docs](https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System#the-javascript-type-language).